source/GameInterface/Services/ItemRosters/Messages/ItemRosterUpdate.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Common.Logging.Attributes;
+using Common.Messaging;
+
+namespace GameInterface.Services.ItemRosters.Messages
+{
+    /// <summary>
+    /// Called when an ItemRoster should be updated.
+    /// </summary>
+    [BatchLogMessage]
+    public class ItemRosterUpdate : ItemRosterMessageBase, ICommand
```

Comment:
For readability and consistency,
```suggestion
    public class UpdateItemRoster : ItemRosterMessageBase, ICommand
```